filecoin-proofs/src/constants.rs

```diff
@@ -47,24 +47,25 @@ pub const PUBLISHED_SECTOR_SIZES: [u64; 10] = [
     SECTOR_SIZE_64_GIB,
 ];
 
+pub struct PorepMinimumChallenges;
+impl PorepMinimumChallenges {
+    pub const fn porep_partitions(sector_size: u64) -> usize {
```

Comment:
I think that a better name for this function would be `from_sector_size`, however to allow for varying PoRep configurations during benchmarking (e.g. `benchy`), I think that it's better to leave this as a mutable `HashMap` rather than hardcoded constants.